Virtusize/Sources/UI/VirtusizeWebViewController.swift

```diff
@@ -192,6 +196,11 @@ extension VirtusizeWebViewController: WKNavigationDelegate, WKUIDelegate {
 				return nil
 			}
 		}
+
+		if VirtusizeAuth.isSNSAuth(viewController: self, webView: webView, url: url) {
```

Comment:
For the detail of the isSNSAuth function, please refer to https://github.com/virtusize/virtusize-auth-ios/pull/1